### ISSUE TITLE
Raid Milestone not being marked as completed.

### ISSUE THREAD

**Issue by iBotPeaches** [Bungie-net/api#158]
So I have other reports of this, but its hard to take people's word in API situations. So I replicated this myself. Note this only became a problem since this reset (week of Sept 19). This was working during last week, so I imagine the changed order encounter probably changed some `hash` of this raid, but that was never pushed into the Milestone.

Since Activity/Stats are still offline I have no other way to check this.

I completed raid (fresh start to end) with this instanceId `408525282` for post game carnage report lookup. So I find this character (My Titan) and pull the Character Progression

```
Array
(
    [questHash] => 82550967
    [stepHash] => 0
    [stepObjectives] => Array
        (
        )

    [tracked] => 
    [itemInstanceId] => 0
    [completed] => 
    [redeemed] => 
    [started] => 
)
```

Info for lookup
```
characterId - 2305843009261041089
membershipId - 4611686018430553461
membershipType - 1
```

So I went digging. The `directorActivityHash` of the raid completed post reset was `2693136602`. The one completed before reset was `2693136601`. I hope this is enough information to confirm.

TLDR - The `directorActivityHash` of the raid changed for this reset period which is not being used in `characterProgressions.milestones` property.

Initial report to me: https://twitter.com/kmhphoto/status/910626015055147008

**Comment by dcaslin** [comment]
Also getting reports of this.

**Comment by GeekyEggo** [comment]
Okay, I'm not going mad (that I know of). I've been trying to figure out _why_ it's indicating incomplete for the past 2 hours. The only workaround I can think of is to check against the activity hash within the variants, and then cross reference with the character activities?

**Comment by dcaslin** [comment]
Yep, that's how I'm working around.

**Comment by GeekyEggo** [comment]
@dcaslin You can ignore my message on the DIM Slack in that case, lol! ;)

**Comment by vthornheart-bng** [comment]
I've not had an opportunity to examine this issue yet, but it's on my radar.  Thanks guys, and sorry for the pain!
